tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -47,6 +47,8 @@ variables:
   MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
   CACHE_KEY: maven | $(Agent.OS) | **/pom.xml, !**/target/**
   CACHE_FALLBACK_KEY: maven | $(Agent.OS)
+  TESTCONTAINER_CACHE_KEY: testcontainer-cache | $(Agent.OS) | **/cache_docker_images.sh
```

Comment:
don't we have to reference `flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/TestContainerVersions.java` here? or am I missing something?

Reply:
Ah, no, we do - I just missed updating the build-apache-repo yml. :facepalm: